sips/sip-015/sip-015-network-upgrade.md

```diff
@@ -0,0 +1,1582 @@
+transaction will default to using Clarity 2 rules.
+
+Due to the introduction of new reserved keywords in Clarity 2, it will not be
+possible to publish an already-written contract as a Clarity 2 contract if it
+uses these reserved words for any other purpose.  However, already-deployed
+contracts are not affected -- they were published under Clarity 1 rules, and
+will continue to be usable as-is.
+
+The read-only contract calls in `pox` will continue to be available to
+contracts.  However, it will no longer be possible to call public functions in
+`pox`.  All future Stacking operations must be done through `pox-2`.
+
+# Activation 
+
+Because this SIP proposes a breaking change it is of paramount importance that
+the ecosystem find this proposal acceptable.  To measure this, three sets of criteria
+must be met in order to determine that there is sufficient support for
+this SIP: one set for Stacked STX holders, one set for un-Stacked STX
+holders, and one set for miners.  These critera are meant to broaden the set of
+participating users above and beyond what has been done in the past.  In
+particular, users who do not Stack are invited to vote on this SIP, as ar users
+who Stack in pools.
+
+In all cases, voting will take place during reward cycles 46 and 47.  This
+window is estimated to begin **starting November 10, 20222** and **ending
+December 8, 2022**.
+
+## For Stackers
+
+In order for this SIP to activate, the following criteria must be met by the set
+of Stacked STX:
+
+* At least 80 million Stacked STX must vote _at all_ to activate this SIP.  This
+  number is chosen because it is more than double the amount of STX locked by
+the largest Stacker at the time of this writing (reward cycle 44).
+
+* Of the Stacked STX that vote, at least 80% of them must vote "yes."
+
+The act of not voting is the act of siding with the outcome, whatever it may be.
+We believe that these thresholds are sufficient to demonstrate interest from
+Stackers -- Stacks users who have a long-term interest in the Stacks
+blockchain's succesful operation -- in performing this upgrade.
+
+### How To Vote
+
+If a user is Stacking, then their STX can be used to vote in one of two ways,
+depending on whether or not they are solo-stacking or stacking through a
+delegate.
+
+The reason voting is streched across two reward cycles is to accommodate users
+whose STX will have unlocked during the voting period.  If a user's STX
+unlock in the first cycle, they can re-Stack them and vote in the second cycle.
+
+In all cases, if a user votes at all, they can only vote in _one_ of the two reward cycles.
+If a user casts a vote in both reward cycles, even if it is for the same
+decision, then the vote is discarded.
+
+#### Solo Stacking
+
+The user must send a minimal amount of BTC from their PoX reward address to one
+of the following Bitcoin addresses:
+
+* For "yes", the address is `11111111111111X6zHB1ZC2FmtnqJ`.  This is the
```

Comment:
Idiot-proofing this - are these the actual voting addresses or just examples? 

Not just this one, but all the ones that follow - I can foresee someone skimming this document and sending their votes to the wrong address if this is not made explicit that these are example addresses and not to be used for activation.

Reply:
Sure -- I bolded the words "yes" and "no" next to each address